bigtable/use-cases/fraudDetection/README.md

```diff
@@ -84,4 +84,54 @@ The current Cloud Bigtable instance configuration does not provide any replicati
+3) Set these environment variables:
+```
+export GOOGLE_APPLICATION_CREDENTIALS={YOUR CREDS PATH}
+export PROJECT_ID={YOUR PROJECT ID}
+```
+
+### Running steps
+
+**1) Build the infrastructure**
+```
+cd terraform
+terraform init
+terraform apply -var="project_id=$PROJECT_ID"
+```
+This builds the infrastructure shown above, populates Cloud Bigtable with customers’ demographics data, and populates Cloud Bigtable with customers’ historical data. It takes about 5-10 minutes to finish.
+It builds the following resources:
+
+| Resource                         | Resource Name                                                                                                                                                                                                 |
+|----------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Cloud Bigtable Instance          | featurestore-{RANDOM\_ID}                                                                                                                                                                                     |
+| Cloud Bigtable Table             | customer-information-{RANDOM\_ID}                                                                                                                                                                             |
+| Cloud Bigtable Column Family     | demographics, history                                                                                                                                                                                         |
+| Cloud Pubsub Input Topic         | transaction-stream-{RANDOM\_ID}                                                                                                                                                                               |
+| Cloud Pubsub Output Topic        | fraud-result-stream-{RANDOM\_ID}                                                                                                                                                                              |
+| Cloud Pubsub Output Subscription | fraud-result-stream-subscription-{RANDOM\_ID}                                                                                                                                                                 |
+| Google Storage Bucket            | fraud-detection-{RANDOM\_ID}                                                                                                                                                                                  |
+| Google Storage Objects           | 1. temp/ directory (for temporary dataflow generated files) <br/> 2. testing_dataset/ directory <br/>3. training_dataset/ directory <br/> 4. ml_model/ directory                                              |
```

Comment:
(nits)
- can we switch from numbers to bullets here.
- i think we can drop the "directory" strings here
- Also can be italicize the stuff inside the brackets for the first item _(for temporary dataflow generated files)_